solr/contrib/ltr/src/java/org/apache/solr/ltr/model/LambdaMARTModel.java

```diff
@@ -55,9 +109,9 @@ public float score(float[] featureVector) {
       }
 
       if (featureVector[featureIndex] <= threshold) {
-        return left.score(featureVector);
+        return (left == null ? 0f : left.score(featureVector));
```

Comment:
minor: are you sure about the style? I usually see this written as: 
`(left == null)? 0f : left.score(featureVector);`